packages/mobile/src/components/lineup/Lineup.tsx

```diff
@@ -242,11 +242,11 @@ export const Lineup = ({
   const handleRefresh = useCallback(() => {
     if (!refreshing) {
       dispatch(
-        actions.refreshInView(true, fetchPayload, limit, extraFetchOptions)
+        actions.refreshInView(true, fetchPayload, null, extraFetchOptions)
```

Comment:
interesting, why?

Reply:
the default limit is "Infinity" which caused issues for profile tracks for some reason. I tested this worked with the lineups that are refresheable